tests/highContrastTests.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+var fluid = require("infusion");
+var jqUnit = fluid.require("node-jqunit", require, "jqUnit");
+var gpii = fluid.registerNamespace("gpii");
+
+require("../extension/src/lib/highContrast.js");
+
+jqUnit.module("GPII Chrome Extension highContrast unit tests");
+
+jqUnit.test("Running unit tests for highContrast", function () {
+
+    var highContrast = gpii.chrome.highContrast();
+
+    jqUnit.assertFalse("highContrastEnabled must be false at the beginning",
+                        highContrast.model.highContrastEnabled);
+    jqUnit.assertEquals("highContrastTheme must be undefined at the beginning",
```

Comment:
You can use [`jqUnit.assertUndefined`](https://github.com/fluid-project/infusion/blob/master/tests/test-core/jqUnit/js/jqUnit.js#L150)